src/documents/adding.md

```diff
@@ -279,17 +194,321 @@ Your flying car has an auto-pilot feature, and you want to add a preference for
+                                        onOff: {
+                                            "true": "On",
+                                            "false": "Off"
+                                        },
+                                        language: "English",
+                                        contrast:{
+                                            "default": "Original",
+                                            "bw": "Black/White",
+                                            "wb": "White/Black"
+                                        },
+                                        autopilot:{
+                                            "value1": "Value 1",
+                                            "value2": "Value 2",
+                                            "value3": "Value 3"
+                                        }
+                                    }
+                                },
+                                "fr-FR": {
+                                    labels: {
+                                        //more content not shown here
+                                    },
+                                    values: {
+                                        onOff: {
+                                            "true": "Oui",
+                                            "false": "Non"
+                                        },
+                                        language: "Français",
+                                        contrast: {
+                                            "default": "Original",
+                                            "bw": "Noir/Blanc",
+                                            "wb": "Blanc/Noir"
+                                        },
+                                        autopilot:{
+                                            "value1": "Valeur 1",
+                                            "value2": "Valeur 2",
+                                            "value3": "Valeur 3"
+                                        }
+                                    }
+                                },
+                                
+                                //more content not shown here
+                }
+    });
+    ```
+
+8. Edit `src/js.panels.js` to assign the selectors equal to the friendly names.
 
+    A. In the `"fluid.defaults()"` call to `"gpii.firstDiscovery.panel.confirm"`, find the `"updatePreferenceValues()"` function.
+    
+    B. Inside the `"updatePreferenceValues()"` function, following the pattern of the other preferences, add the code to change the "Autopilot" label selector, `"autopilotLabel"`, to use the friendly name assigned in the previous step. 
+    
+    ```javascript
+    gpii.firstDiscovery.panel.confirm.updatePreferenceValues = function(that, changeContext) {
+    
+            //more content not shown here
+            
+            var language = changeContext.value.gpii_firstDiscovery_language;
```

Comment:
We should avoid repetitive manual binding code of this sort - see comments on the main pull https://github.com/GPII/first-discovery/pull/108 and work by @benjaminstokes on adding sufficient features to @the-t-in-rtf gpii-binder discussion at https://botbot.me/freenode/fluid-work/2016-03-01/?msg=61284220&page=2